src/geoserverexplorer/gui/dialogs/styledialog.py

```diff
@@ -107,10 +107,10 @@ def __init__(self, catalog, layer, parent = None):
         styles = layer.styles
         self.layerstyles = [style.name for style in styles]
         self.layerdefaultstyle = layer.default_style.name
-            if layer.default_style is not None else ''
-        self.style = None
-        self.default = None
-        self.initGui()        
+        if layer.default_style is not None:
+            self.style = None
+            self.default = None
+            self.initGui()        
         
```

Comment:
This doesn't look like a correct fix. It should all go in one line (self.layerdefaultstyle = layer.default_style.name if layer.default_style is not None else '')

With your change, the initGui method is not called if the default style is None, so it wont do anything. Also, in that case, the self.layerdefaultstyle = layer.default_style.name line would raise an exception since None.name is wrong